### ISSUE TITLE
More conventional setup process

### ISSUE THREAD

**Issue by marshall007** [superfly/cdn#19]
The recommended [setup process](https://github.com/superfly/cdn/blob/ea338cff0538cf7e07b0d9e5cc733d5c1ffdcde4/README.md#try-the-starter-app) is rather unconventional and difficult to recall without referencing the README. I think a better approach would be for the `fly` CLI to have a built in generator replicating the project structure in the [referenced gist](https://gist.github.com/mrkurt/ebc48856b74fde392a6d62a032b59a97).

Setup might then look something like:

```sh
# create fly app, project folder, and `.fly.yml` config
# walk user through setup (similar to `npm init`):
#  - name: example-app
#  - eject (Y/n): y
#  - typescript (Y/n): y
#  - ...
fly apps create
```

If the user chose not to "eject" their fly app configuration into code upon initial setup, that can still be done later at anytime (prior art: Angular's `ng eject`):

```sh
# code generation based on current `.fly.yml` config
# create webpack, tsconfig, etc if they don't exist (error if they do)
# create/update `package.json` (adding `@fly/cdn` dependency)
# remove `flyCDN` config from `.fly.yml`
fly apps eject --app <name>
```

Another option, upon ejecting the config, would be to store fly-related configuration in the `package.json` and eliminate `.fly.yml` entirely. As an example, `babel` allows you to do this [in place of a `.babelrc.js`](https://babeljs.io/docs/en/configuration#packagejson)). Would be nice because things like `name` could be shared between the two.

**Comment by mrkurt** [comment]
Yeah I would like to have an app generator. I really like `create-react-app`.

**Comment by michaeldwan** [comment]
The [cli's app generator](https://github.com/superfly/fly/blob/master/packages/core/src/cmd/new.ts#L22) could do this. Right now it's giving the user a choice to clone an app from the examples package, but maybe it'd be better to remove the multiple choice and clone a fully configured cdn app instead.

**Comment by michaeldwan** [comment]
To be clear it's not cloning as in `git clone`, just copying the directory structure from the examples package to the target directory.
